Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchyBasedLinkResolver.swift

```diff
@@ -157,15 +157,15 @@ final class PathHierarchyBasedLinkResolver {
         }
     }
     
-    /// Adds a technology and its volumes and chapters to the path hierarchy.
-    func addTechnology(_ technology: DocumentationContext.SemanticResult<TutorialTableOfContents>) {
-        let reference = technology.topicGraphNode.reference
+    /// Adds a tutorial table-of-contents page and its volumes and chapters to the path hierarchy.
+    func addTutorialTableOfContents(_ tutorialTableOfContents: DocumentationContext.SemanticResult<TutorialTableOfContents>) {
+        let reference = tutorialTableOfContents.topicGraphNode.reference
 
-        let technologyID = pathHierarchy.addTutorialOverview(name: linkName(filename: technology.source.deletingPathExtension().lastPathComponent))
+        let technologyID = pathHierarchy.addTutorialOverview(name: linkName(filename: tutorialTableOfContents.source.deletingPathExtension().lastPathComponent))
         resolvedReferenceMap[technologyID] = reference
```

Comment:
I suggest not using technology here either:
```suggestion
        let tutorialTableOfContentsID = pathHierarchy.addTutorialOverview(name: linkName(filename: tutorialTableOfContents.source.deletingPathExtension().lastPathComponent))
        resolvedReferenceMap[tutorialTableOfContentsID] = reference
```

Reply:
Updated in https://github.com/swiftlang/swift-docc/pull/1039/commits/44fece98cd31014ac38d962fbe0993358111d0bc